test/rules-custom.spec.ts

```diff
@@ -0,0 +1,61 @@
+import 'mocha';
```

Comment:
I added this spec file as a place where we can test edge cases in a more end-to-end fashion which aren't exposed by out-of-the-box rulesets. Interested if there are other recommendations for where/how to test edge cases like these.